vagabond/app/views/users/new.html.erb

```diff
@@ -1,20 +1,27 @@
-Sign Up
+<h2 style="text-align:center"><strong>Sign Up</strong></h2>
 
-<%= form_for @user do |f| %>
-  <div>
-    <%= f.text_field :first_name, placeholder: "First Name" %>
-  </div>
-  <div>
-    <%= f.text_field :last_name, placeholder: "Last Name" %>
-  </div>
-  <div>
-    <%= f.text_field :current_city, placeholder: "Current City" %>
-  </div>
-  <div>
-    <%= f.text_field :email, placeholder: "Email" %>
-  </div>
-  <div>
-    <%= f.password_field :password, placeholder: "Password" %>
-  </div>
-  <%= f.submit "Sign Up" %>
-<% end %>
+<div class="register">
+
+  <%= form_for @user, html: { multipart: true } do |f| %>
+    <div class="form-group">
```

Comment:
Why does only this have a class of `form-group` and the others have a class of `register-input`?

getbootstrap.com/css/#forms